stable_ingress_egress/stable_gen/Readme.md

```diff
@@ -0,0 +1,65 @@
+## Test suite for Kubernetes
```

Comment:
stable_gen ... is this another redundant folder ... can you remove all files form this folder ?